### PR TITLE
Creating a blank disk image (DV/PVC)

### PR DESCRIPTION
**Pull request by ousleyp**
@openshift/team-documentation PTAL <3

### CODE REVIEW

**Comment by mburke5678** [comment]
@ousleyp A couple of pesky questions. Otherwise LGTM

**Comment by ousleyp** [comment]
> @ousleyp A couple of pesky questions. Otherwise LGTM

Where can I find the questions? :)

**Comment by mburke5678** [comment]
@ousleyp Sorry. I think I added the comments. i was commenting via my smart phone for the first time. Apparently I didn't complete the process.

File: modules/cnv_blankdisk.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+=== Creating a blank disk image in a PVC
+
+You can create a new blank disk image in a *PersistentVolumeClaim* with 
+a DataVolume or PVC manifest. You can use blank disks to increase your storage 
+capacity or create new data partitions.
+
+.Prerequisites
+
+* {ProductName} 1.4
+* At least one available PV
```
[the remark this answers is not in the document]
Comment (ousleyp) [reply modules/cnv_blankdisk.adoc]: I'll spell it out - good call :)

File: modules/cnv_blankdisk.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+.Procedure
+
+. Create the manifest file.
+* To use a DataVolume manifest, create `blank-image-datavolume.yaml`:
```
[the remark this answers is not in the document]
Comment (ousleyp) [reply modules/cnv_blankdisk.adoc]: I don't think it's required (on either count). I'll take the filename out of the first step.

**Comment by ousleyp** [comment]
> @ousleyp Sorry. I think I added the comments. i was commenting via my smart phone for the first time. Apparently I didn't complete the process.

Thanks for the review, mburke! No worries at all. Please check out the changes I made and let me know what you think. I also split it up into two procedures based on other feedback.

**Pull request merged**